app/controllers/redirected_routes/url_handler.rb

```diff
@@ -14,7 +14,8 @@ class RedirectedRoutes::UrlHandler
     teams_show: /(^\/teams\/\d+)$/,
     folder_show: /(^\/teams\/\d+\/folders\/\d+)$/,
     accounts_show: /(^\/accounts\/\d+)$/,
-    profile: /(^\/profile)$/
+    profile_index: /(^\/profile)$/,
+    admin_users_index: /(^\/admin\/users)$/
```

Comment:
for now this is the way to go.
keep in mind to change this as soon we have migrated most of the UI stuff to ember.
then we probably can work with a more generic approach. (redirect unless /api/*)